### PR TITLE
Release staging to prod

### PR DESCRIPTION
**Pull request by Br3nda**

### CODE REVIEW

**Comment by KatieWright26** [comment]
This looks good to go - the bug fixes I made previously appear to be missing from staging though - primarily the [vocab print view](https://github.com/ODNZSL/nzsl-online/pull/130) and the [dropdown search bug](https://github.com/ODNZSL/nzsl-online/pull/135)

File: app/assets/stylesheets/_searchbar.scss [hunk 1]
```diff
@@ -151,7 +151,7 @@
 }
 
 .locations .dropdown {
-  width: 400px;
+  width: 430px;
```
Comment (Br3nda) [inline app/assets/stylesheets/_searchbar.scss]: @katie is this the change you you made that you are saying isn't on staging?

**Comment by Br3nda** [comment]
Looks like an ` bundle exec rake assets:precompile` needs adding to the deploy procedure, then @KatieWright26's recent css changes will take effect

**Comment by Br3nda** [comment]
I don't have access to prod or the deploy config. @katiewright26 you'll need to have love a look

**Comment by Br3nda** [comment]
I can't "request changes" my own pull request so leaving a big giant comment here..  Don't merge this until the asset precompile in deployment is fixed. Otherwise we'd be deploying code that could precompiled into an untested state at any moment.

**Comment by KatieWright26** [comment]
@Br3nda re precompiling, are you needing me to run a rails console for the prod app and run `bundle exec rake assets:precompile`, then merge this PR?

**Comment by Br3nda** [comment]
No, the deploy to heroku needs modifying to include running asset precompile. I dont have access to prod on the heroku compile so I can't reconfigure it.

**Comment by elspeth-rabid** [comment]
Ah cool! I can help with that :-)

**Comment by Br3nda** [comment]
Iirc it currently uses heroku pipelines. But it doesn't have to. Travis with encrypted keys can work for deploy

**Comment by KatieWright26** [comment]
@Br3nda, @elspeth-rabid and I couldn't find a way to add the precompile command to the heroku pipeline, but staging looks like it is behaving as expected. This PR should be good to be merged to prod.

**Pull request merged**